test/components/FeatureFlagRenderer.test.js

```diff
@@ -61,7 +63,7 @@ describe("components/FeatureFlagRenderer", () => {
           />
         );
         wrapper.setState({ flagValue: true });
-        expect(wrapper.text()).toEqual(renderFeatureCallback());
+        expect(wrapper.text()).toEqual(renderFeatureCallback().props.children);
```

Comment:
So doing this doesn't work because the wrapper here is the entire shallow wrapper of the FeatureFlagRenderer component while `renderFeatureCallback()` returns just the react div element with "feature rendered" as a child.